frontend/src/views/ZimcheckDashboard.vue

```diff
@@ -0,0 +1,197 @@
+          v-for="(errors, scraper) in checkData"
+          id="hide"
+          :key="scraper"
+        >
+          <td>
+            {{ scraper }}
+          </td>
+          <td>
+            <span v-if="checkTotals.integrity !== 0">
+              {{ Math.round((errors.integrity * 100) / checkTotals.integrity) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.empty !== 0">
+              {{ Math.round((errors.empty * 100) / checkTotals.empty) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.metadata !== 0">
+              {{ Math.round((errors.metadata * 100) / checkTotals.metadata) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.favicon !== 0">
+              {{ Math.round((errors.favicon * 100) / checkTotals.favicon) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.main_page !== 0">
+              {{ Math.round((errors.main_page * 100) / checkTotals.main_page) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.redundant !== 0">
+              {{ Math.round((errors.redundant * 100) / checkTotals.redundant) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.url_internal !== 0">
+              {{ Math.round((errors.url_internal * 100) / checkTotals.url_internal) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.url_external !== 0">
+              {{ Math.round((errors.url_external * 100) / checkTotals.url_external) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+          <td>
+            <span v-if="checkTotals.redirect !== 0">
+              {{ Math.round((errors.redirect * 100) / checkTotals.redirect) }}
+            </span>
+            <span v-else>0</span> %
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <ErrorMessage
+      v-if="error"
+      :message="error"
+    />
+  </div>
+</template>
+
+<script>
+import Common from '../Common.mixin.js'
+import ErrorMessage from '../components/ErrorMessage.vue'
+
+export default {
+  name: 'ZimcheckDashboard',
+  components: {
+    ErrorMessage
+  },
+  mixins: [Common],
+  data () {
+    return {
+      checkData: null,
+      checkTotals: null,
+      error: null // error message generated by API
+    }
+  },
+  created () {
+    const parent = this
+    this.startLoading()
+
+    const url = '/dashboard'
+
+    this.queryAPI('GET', url)
+      .then(function (response) {
+        parent.checkData = response.data.checkData
+        parent.checkTotals = response.data.checkTotals
```

Comment:
Here's a hint on how to get a single, scraper-aware dict from the API as well as having an extendable list of checks

```js
        // list of known check we want to display values for (can be extended)
        const knownChecks = new Map([
          ['checksum', 'error'],
          ['integrity', 'error'],
          ['empty', 'error'],
          ['metadata', 'error'],
          ['favicon', 'error'],
          ['main_page', 'error'],
          ['redundant', 'warning'],
          ['url_internal', 'error'],
          ['url_external', 'error'],
          ['redirect', 'error']
        ])

        parent.checkTotals = new Map()
        knownChecks.forEach((value, key) => parent.checkTotals.set(key, 0))

        parent.scraperData = response.data.checkData
        for (const data of Object.values(scraperData)) {
          for (const [check, value] of Object.entries(data)) {
            parent.checkTotals.set(check, (parent.checkTotals.get(check) || 0) + value)
          }
        }
```